### PR TITLE
add: bopl-battle (BepInEx)

### PR DESCRIPTION
**Pull request by ethangreen-dev**

### CODE REVIEW

File: src/model/game/GameManager.ts [hunk 1]
```diff
@@ -601,6 +601,13 @@ export default class GameManager {
             "https://thunderstore.io/c/balatro/api/v1/package/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2379780")], "Balatro.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.LOVELY, []),
+
+        new Game(
+            "Bopl Battle", "BoplBattle", "BoplBattle",
+            "Bopl Battle", ["BoplBattle.exe"], "BoplBattle_Data",
+            "https://thunderstore.io/c/palworld/api/v1/package/", EXCLUSIONS,
```
Comment (anttimaki) [inline src/model/game/GameManager.ts:608]: The URL should be updated.

**Review by anttimaki — CHANGES_REQUESTED** [review]
Copy-pasted API endpoint, otherwise LGTM.

File: src/model/game/GameManager.ts (hunk 1 above)
Comment (ethangreen-dev) [reply src/model/game/GameManager.ts]: Thanks for catching that, fixed.

**Review by anttimaki — APPROVED** [review]

**Comment by mathlover3** [comment]
how long will it be untill its added?

**Comment by AbstractMelon** [comment]
> how long will it be untill its added?

Patience david

**Comment by anttimaki** [comment]
Closing in favor of #1301 which includes these changes too.

**Pull request closed without merging**